server/lib/ga-collector.js

```diff
@@ -65,7 +65,7 @@ GACollector.prototype = {
    * @param {Object} body
    */
   write: function (body) {
-    if (! body || ! body.events || ! body.events.length > 0 || ! this.analyticsId) {
+    if (! body || ! body.events || body.events.length === 0 || ! this.analyticsId) {
```

Comment:
In classic JS fashion, I think this one was actually behaving as intended before, even though it made an incorrect assumption about the relative precedence of the `!` and `>` operators:

* If `length` is greater than zero, `! length` is false, which coerces to zero, which doesn't enter the condition.

* If `length` is zero, `! length` is true`, which coerces to one, which does enter the condition.

Good old JS, it's always got your back.